apps/microsoft-teams/frontend/src/components/config/ChannelSelection/ChannelSelection.tsx

```diff
@@ -0,0 +1,73 @@
+import { Box, Flex, IconButton, ModalLauncher, Text, TextInput } from '@contentful/f36-components';
+import AddButton from '@components/config/AddButton/AddButton';
+import ChannelSelectionModal from '@components/config/ChannelSelectionModal/ChannelSelectionModal';
+import TeamsLogo from '@components/config/TeamsLogo/TeamsLogo';
+import { channelSelection } from '@constants/configCopy';
+import { Notification } from '@customTypes/configPage';
+import { EditIcon } from '@contentful/f36-icons';
+import { styles } from './ChannelSelection.styles';
+// TODO: update this when we start fetching channel installations
+import mockChannels from '@test/mocks/mockChannels.json';
+
+interface Props {
+  notification: Notification;
+  handleNotificationEdit: (notificationEdit: Partial<Notification>) => void;
+}
+
+const ChannelSelection = (props: Props) => {
+  const { notification, handleNotificationEdit } = props;
+
+  const openChannelSelectionModal = () => {
+    return ModalLauncher.open(({ isShown, onClose }) => (
+      <ChannelSelectionModal
+        isShown={isShown}
+        onClose={() => {
+          onClose(true);
+        }}
+        handleNotificationEdit={handleNotificationEdit}
+        savedChannelId={notification.channelId}
+      />
+    ));
+  };
+
+  // TODO: update this when we start fetching channel installations
+  const getChannelName = (channelId: string) => {
+    const channel = mockChannels.find((channel) => channelId === channel.id);
+    const displayName = channel ? `${channel.name}, ${channel.teamName}` : '';
```

Comment:
hm i wonder if silently swallowing the case where we don't find a missing channel is a good idea 🤔 

i don't need know that we need to error per se but what if we put `'unknown'` or `'channel not found'` here in the meantime?

Reply:
Agreed, definitely something we will want to update once we are actually fetching the channel installations. This was feeling very temporary to me, but I can add in some copy to display in the meantime in a follow up!